packages/contracts/test/transactionManager.spec.ts

```diff
@@ -81,16 +82,16 @@ describe("TransactionManager", function () {
       receivingAddress: receiver.address,
       callData: "0x",
       transactionId: hexlify(randomBytes(32)),
-      sendingChainId: 1337,
-      receivingChainId: 1338,
+      sendingChainId: (await transactionManager.chainId()).toNumber(),
+      receivingChainId: (await transactionManagerReceiverSide.chainId()).toNumber(),
```

Comment:
We should rely on these and remove unnecessary overrides within the tests themselves

Reply:
gotchaa.